content/md/en/docs/reference/substrate-connect.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Substrate Connect
+description: Run Wasm Light Clients of any Substrate based chain directly in your browser
+keywords:
+---
+Substrate is a modular framework for creating use-case optimized blockchains at a low cost, by composing custom or pre-built components. Substrate is the backbone that powers Polkadot, a next-generation, heterogeneous, multi-chain network, and its ecosystem.
+
+## Wasm Light Clients
+
+Substrate-connect turns a browser into a node on a network allowing end-users of Web3 apps to interact with blockchains directly - without connecting to third-party remote nodes and other servers. Removing intermediary servers between network and its users improves security, simplifies infrastructure of a network and lowers its maintenance costs. Decentralized in-browser light clients are a unique feature of substrate-based networks.
+
+[What is a light client and why you should care? by Thibault Sardan](https://www.parity.io/what-is-a-light-client/)
+
```

Comment:
There are 3 things about substrate-connect.
A) Install `@polkadot/rpc-provider` that contains an rpc provider of substrate-connect. With this installation, a user can out-of-the-box use that implementation into migrating from existing use of polkadotJS api to having a light client integrated. (best for Dapps that already use PolkadotJS api, or devs who want to get an app, up and running in no-time)
B) Install `@substrate/connect` this is the actual npm package, that needs severe development in order to work - meaning there is specific handling of sequence of calls that needs to be done (e.g. add relay/wellKnown chain - add parachain in that order etc), handle the JsonRpcCallbacks etc etc (best for adding light client in extensions, desktop apps and/or creating library on top of Substrate/connect)
C) Substrate connect Extension. Extension is actually a "helper" (if I may say), of the dApps that having already integrated substrate connect (Case A above). Extension runs the light client when browser starts, and syncs everything (all well known chains), before a DApp inits. Once a Dapp that contains substrate-connect inits, then Substrate-Connect "understands" the existence of the Extension and uses the already-synced light client of the extension instead of firing up a new one inside the Dapp's tab.

With the above I covered the following topics:
> Install substrate-connect 

Options A and B above depending on the needs of the developer

> Install browser extension

Option C above

> Connect to a well-known blockchain

This is covered in the `content/md/en/docs/reference/substrate-connect.md` file. There are example for either simple usage or advanced usage on how to connect to a well-known chain

> Integrate in apps that use Polkadot-JS API

At the same file as mentioned above, this is covered on paragraph: `Usage with PolkadotJS API (PolkadotJS provider)`

> Add Substrate Connect to other applications

Same answer as above (simple usage, advanced usage)